django/db/backends/mysql/introspection.py

```diff
@@ -257,6 +258,10 @@ def get_constraints(self, cursor, table_name):
                 information_schema.table_constraints AS c
             WHERE
                 kc.table_schema = DATABASE() AND
+                (
+                    kc.referenced_table_schema = DATABASE() OR
+                    kc.referenced_table_schema IS NULL
+                ) AND
```

Comment:
This is untested.

Reply:
5a3074ab372b43170933736712608bbdc0675764